### PR TITLE
Add installation guide and badges to README

### PR DESCRIPTION
**Pull request by mkj-is**
Screenshot 2019-03-28 at 13 50 40

### CODE REVIEW

**Review by mstana — DISMISSED** [review]

**Review by mstana — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -9,3 +12,11 @@ FuntastyKit iOS framework contains:
 - Some regularly used UIKit extensions
 - Protocols for simple initialization from XIB files, storyboards and for handling keyboard
 - Hairline constraint for one-pixel designs
+
+## Installation
+
+If use use CocoaPods for dependency management add following line to your `Podfile`:
```
Comment (tomasbabulak) [inline README.md]: use use 😱

Comment (tomasbabulak) [inline README.md]:
```suggestion
If you use CocoaPods for dependency management, add following line to your `Podfile`:
```

File: README.md [hunk 2]
```diff
@@ -9,3 +12,11 @@ FuntastyKit iOS framework contains:
 - Some regularly used UIKit extensions
 - Protocols for simple initialization from XIB files, storyboards and for handling keyboard
 - Hairline constraint for one-pixel designs
+
+## Installation
+
+If use CocoaPods for dependency management add following line to your `Podfile`:
```
Comment (tomasbabulak) [inline README.md]:
```suggestion
If you want to use CocoaPods for dependency management, add following line to your `Podfile`:
```

**Review by tomasbabulak — APPROVED** [review]
👏

**Pull request merged**